tensorly/tests/test_backend.py

```diff
@@ -308,3 +308,75 @@ def test_partial_svd():
     with T.assert_raises(ValueError):
         tensor = T.tensor(np.random.random((3, 3, 3)))
         T.partial_svd(tensor)
+
+
+def test_shape():
+    A = T.arange(3*4*5)
+
+    shape1 = (3*4,5)
+    A1 = T.reshape(A, shape1)
+    T.assert_equal(T.shape(A1), shape1)
+
+    shape2 = (3,4,5)
+    A2 = T.reshape(A, shape2)
+    T.assert_equal(T.shape(A2), shape2)
+
+
+def test_ndim():
+    A = T.arange(3*4*5)
+    T.assert_equal(T.ndim(A), 1)
+
+    shape1 = (3*4,5)
+    A1 = T.reshape(A, shape1)
+    T.assert_equal(T.ndim(A1), 2)
+
+    shape2 = (3,4,5)
+    A2 = T.reshape(A, shape2)
+    T.assert_equal(T.ndim(A2), 3)
```

Comment:
Not sure tests are needed for `shape` and `ndim` since we don't actually implement these ourselves but I guess it doens't hurt..

Reply:
This might be left over from when I was modifying the behavior of `reshape()`. I can remove.